charts/argo-rollouts/values.yaml

```diff
@@ -53,6 +53,8 @@ controller:
   deploymentLabels: {}
   # -- Annotations to be added to application controller pods
   podAnnotations: {}
+  # Labels to be added to the application controller pods
```

Comment:
Ah, I think we need `--` since the comment is not shown in README.md 👀 

```diff
-# Labels to be added to the application controller pods
+# -- Labels to be added to the application controller pods
```